tests/test_namelist.py

```diff
@@ -151,6 +175,40 @@ def test_default_config_has_entries(self):
         for name, value in config.items():
             self.assertIsInstance(name, str, f'key {name} is not a string')
 
+    def test_enable_restart_from_default(self):
```

Comment:
Could you split these tests off into another TestCase class? They don't use the setup or teardown so it won't break the code, and I think it would help better organize the tests. The only reason I'm asking is for test organization (I find it's nice to have the tests that test one feature in one TestCase). Feel free to disagree.

Reply:
These tests were introduced in a previous PR (#9) so don't know why they are showing up here. I think this is a GitHub issue. Still, I agree with your point. I created an issue for this and the enable_restart documentation point. See #24

Reply:
Great.